SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1842,12 +1852,21 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			RPCService->AdvanceView();
 		}
 
+		if (DebugCtx != nullptr)
+		{
+			DebugCtx->AdvanceView();
```

Comment:
nit: advance and advanceView are the same, could we be consistent with function names in subviews?

Reply:
in this case it actually isn't the same! `AdvanceView` calls (both here and the RPC service one) only update their local views, and run no logic. An `Advance` on the other hand (in the case of the actor system and well known entity system) may update state and run logic.